src/pages/billing/Project.js

```diff
@@ -6,6 +6,7 @@ import { ButtonPrimary, IdContainer, Link, Select, SimpleTabBar, spinnerOverlay
 import { DeleteUserModal, EditUserModal, MemberCard, NewUserCard, NewUserModal } from 'src/components/group-common'
 import { icon, spinner } from 'src/components/icons'
 import Modal from 'src/components/Modal'
+import { MiniSortable } from 'src/components/table'
```

Comment:
I'm almost inclined to pull this out into its own file, since nothing stops us from sorting non-table things with this, but wanted to open for discussion.